src/components/StandingsList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import standingData from '../../data/get-standings.json';
+import Standings from './Standings';
+
+function StandingsList() {
+  const standings = standingData.response[0].league.standings[0]; // Note the correct property name here
```

Comment:
We don't have to address this here (maybe you can raise an issue for it and we'll do it later), but I want to respond to this because I think you mentioned it in your PR overview:

This very long chain of properties to get the data you need is ugly (it doesn't look nice) brittle (it can easily go wrong ) and unintuitive (it's impossible to guess and hard to lookup what this path might be).

Usually what we'd do to solve this is TRANSFORM the original response data we get from the server. We'd do this in our request script before we save it to JSON (or otherwise send it on to the app).

The transformer function needs to anticipate the data needed by the app - which in this case, is just the premier league table.

So basically what you do is:
* Fetch the data
* Pull out `response[0].league.standings[0]` from the response data into a variable
* Save the extracted data into your JSON file.

The transformer may also remove unused data or change property names.

Once your data has been nicely processed, it's much easier, more intuitive and more robust to use in the frontend.